docs/examples/userguide/parallelism/cimport_openmp.py

```diff
@@ -0,0 +1,11 @@
+# tag: openmp
+
+from cython.parallel import parallel
+from cython.cimports.openmp import omp_set_dynamic
+
+num_threads = cython.declare(cython.int)
+
+omp_set_dynamic(1)
+with cython.nogil, parallel():
+    num_threads = openmp.omp_get_num_threads()
```

Comment:
Presumably this needs to be part of ` from cython.cimports.openmp import omp_set_dynamic`

Reply:
Hmm all examples I did check via compiling. Maybe I did copy-past error. Let me check

Reply:
Issue fixed. File now can be compiled by cython.